packages/web-twig/src/Resources/components/FileUploader/FileUploaderInput.twig

```diff
@@ -0,0 +1,90 @@
+
+{# Class names #}
+{%- set _rootClassName = _spiritClassPrefix ~ 'FileUploaderInput' -%}
+{%- set _dropzoneClassName = _rootClassName ~ '__dropZone' -%}
+{%- set _dropzoneLabelClassName = _rootClassName ~ '__dropZoneLabel' -%}
+{%- set _linkClassName = _rootClassName ~ '__link' -%}
+{%- set _dragAndDropLabelClassName = _rootClassName ~ '__dragAndDropLabel' -%}
+{%- set _inputClassName = _rootClassName ~ '__input' -%}
+{%- set _labelClassName = _rootClassName ~ '__label' -%}
+{%- set _labelHiddenClassName = _isLabelHidden ? _rootClassName ~ '__label--hidden' : null -%}
+{%- set _labelRequiredClassName = _isRequired ? _rootClassName ~ '__label--required' : null -%}
+{%- set _messageClassName = _rootClassName ~ '__message' -%}
+{%- set _rootValidationStateClassName = _validationState ? _rootClassName ~ '--' ~ _validationState : null -%}
+{%- set _helperTextClassName = _rootClassName ~ '__helperText' -%}
+
+{# Attributes #}
+{%- set _requiredAttr = _isRequired ? 'required' : null -%}
+
+{# Miscellaneous #}
+{%- set _styleProps = useStyleProps(props) -%}
+{%- set _allowedAttributes = [ 'accept', 'multiple' ] -%}
+{%- set _labelClassNames = [ _labelClassName, _labelHiddenClassName, _labelRequiredClassName ] -%}
+{%- set _mainPropsWithoutReservedAttributes = props | filter((value, prop) => prop not in ['data-spirit-element', 'id']) -%}
+{%- set _rootClassNames = [ _rootClassName, _rootValidationStateClassName, _styleProps.className ] -%}
+
+<div
+    {{ mainProps(_mainPropsWithoutReservedAttributes) }}
+    {{ classProp(_rootClassNames) }}
+    {{ styleProp(_styleProps) }}
+    data-spirit-element="wrapper"
+>
+    <label for="{{ _id }}" {{ classProp(_labelClassNames) }}>
+        {%- if _unsafeLabel -%}
+            {{ _unsafeLabel | raw }}
+        {%- else -%}
+            {{ _label }}
+        {%- endif -%}
+    </label>
+    <input
+        {{ inputProps(props, _allowedAttributes) }}
+        type="file"
+        id="{{ _id }}"
+        name="{{ _name }}"
+        class="{{ _inputClassName }}"
+        data-spirit-element="input"
+        {{ _requiredAttr }}
+    />
+    <div class="{{ _dropzoneClassName }}" data-spirit-element="dropZone">
+        <Icon name="file" />
+        <label for="{{ _id }}" class="{{ _dropzoneLabelClassName }}">
+            <span class="{{ _linkClassName }} link-primary link-underlined">{{ _pickAFileText }}</span>
+            <span class="{{  _dragAndDropLabelClassName }}">{{ _dragAndDropText }}</span>
+        </label>
+        {% if _helperText and not _unsafeHelperText %}
+            <div class="{{ _helperTextClassName }}">
+                {{ _helperText }}
+            </div>
+        {% endif %}
+        {% if _unsafeHelperText %}
+            <div class="{{ _helperTextClassName }}">
+                {{ _unsafeHelperText | raw }}
+            </div>
+        {% endif %}
+    </div>
+    {% if _message and not _unsafeMessage %}
+        <div class="{{ _messageClassName }}">
+            {{ _message }}
```

Comment:
What about
```suggestion
            {% if _message and not _unsafeMessage %}{{ _message }}{% else %}{{ _unsafeMessage | raw }}{% endif %}
```

or something like that?

Reply:
Or maybe we should just prefer one over another as suggested @dlouhak. Should we prefer safe way over unsafe?